### PR TITLE
Add Alephium Official Wallets

### PR DESCRIPTION
**Pull request by MaudSimon**
Please don't hesitate to correct if I messed-up something.

### CODE REVIEW

File: data/alephium-official-wallet [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "name": "Alephium Official Wallets",
+  "description": "Stay in control of Alephium assets and connect to dApps of the Alephium ecosystem. Designed with simplicity and usability in mind, even complex actions are easy to perform. The Alephium wallets are fully self-custodial, only you have access to your seeds and private keys.",
+  "short_description": "The Alephium wallets are your keys to the Alephium ecosystem",
+  "tags": ["Wallets"],
+  "councils_choice": true,
+  "verified": true,
+  "dotw": false,
+  "links": {
+    "Desktop Wallet": "https://github.com/alephium/alephium-frontend/releases/latest",
+    "Mobile Wallet IOS": "https://apps.apple.com/us/app/alephium-wallet/id6469043072"
```
Comment (Fugashu) [inline data/alephium-official-wallet]: missing a `,`

File: data/alephium-official-wallet [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "Mobile Wallet Android": "https://play.google.com/store/apps/details?id=org.alephium.wallet"
```
Comment (Fugashu) [inline data/alephium-official-wallet]: missing a `,`

File: data/alephium-official-wallet [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "Chrome Browser Extension": "https://chrome.google.com/webstore/detail/alephium-extension-wallet/gdokollfhmnbfckbobkdbakhilldkhcj"
```
Comment (Fugashu) [inline data/alephium-official-wallet]: missing a `,`

File: data/alephium-official-wallet [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (Fugashu) [inline data/alephium-official-wallet]: The whole file should end with the `.json` extension. Yours doesn't have that right now :)

**Review by Fugashu — CHANGES_REQUESTED** [review]

**Comment by MaudSimon** [comment]
@Fugashu should be good now :)

**Pull request merged**